plugins/autoloader/prism-autoloader.js

```diff
@@ -10,8 +10,19 @@
 
 	var ignored_language = 'none';
 
+	var script = document.getElementsByTagName('script');
+	script = script[script.length - 1];
+	var languages_path;
+	if(script.hasAttribute('data-autoloader-path')) {
+		var path = script.getAttribute('data-autoloader-path').trim();
+		if(path != '') {
+			languages_path = path;
+		}
```

Comment:
If this condition isn't met, we'd end up with `languages_path` as `undefined`. Should we have a fall back / default for `languages_path`?